object-search/pods/craft.yml

```diff
@@ -0,0 +1,56 @@
+!CompoundExecutor
+components:
+  - !ImageReader
+    with:
+      from_bytes: True
+    metas:
+      name: img_read
+  - !ImageNormalizer
+    with:
+      target_size: 96
+      img_mean: [0.485, 0.456, 0.406]
+      img_std: [0.229, 0.224, 0.225]
+      channel_axis: -1
+    metas:
+      name: img_norm
+  - !TorchObjectDetectionSegmenter
+    with:
+      channel_axis: -1
+      confidence_threshold: 0.98
+    metas:
+      name: img_detect
+  - !ImageResizer
```

Comment:
Remember that the detected objects will need to be normalized again before encoder step. But at the same time, the documents that need to go to the KVIndexer, do not have to be normalized